packages/stream-text/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "@importantimport/tsconfig/app.json",
+  "files": ["vitest.config.ts"],
```

Comment:
```suggestion
```

Reply:
Modify tsconfig.node.json in the root directory

Reply:
just extends from ../../tsconfig.node.json will do